src/redux/thunks/applyResource.ts

```diff
@@ -36,12 +36,13 @@ function applyKustomization(
   fileMap: FileMapType,
   kubeconfig: string,
   context: string,
+  namespace: string,
```

Comment:
shouldn't this be optional? I'm thinking that in case the kustomization generates resources for multiple namespaces the won't want to specify this at all - since it would force all resources into the same namespace..

Reply:
@olensmar Should I add a third option in the modal ( `Use kustomization namespaces` ) or should we not show the namespace selector at all?

Reply:
perhaps there could be a "<as configured>" option which is selected by default and would result in Monokle not setting the namespace explicitely? (this would apply to all deploy actions)

Reply:
@olensmar wouldn't that be confusing from the UX perspective? The users might think that a namespace is required for deployment.. I think a third radio button would be better, but we need to decide on the label

Reply:
so we have "Use existing namespace", "Create new namespace" and the third one could be "None" ? or "Neither" ?

Reply:
sorry - my bad - "Use Existing" is fine as long as it doesn't override what is in the manifests